src/components/TicTacToe.tsx

```diff
@@ -19,9 +21,16 @@ class TicTacToe extends React.Component<any, any> {
 
   public submitBoardSize = (event: any): void => {
```

Comment:
This can be typed into a form submit type

Reply:
Do you know where these `React.FormEvent` types are documented? I can only ever find them on StackOverflow.

Reply:
I look at the react types file in node modules for it, haha. That's what I use